### PR TITLE
Selector for the Integration

### PR DESCRIPTION
**Pull request by ishwari-gaikwad2**

### CODE REVIEW

File: force-app/main/default/classes/OneJNQuerySelector.cls [hunk 1]
```diff
@@ -13,31 +13,109 @@ public with sharing class OneJNQuerySelector {
    * @description retrieve accounts
    * @author Remario Richards | 05-26-2021
    **/
-  public static void getAccounts() {
```
Comment (RemarioCRM) [inline force-app/main/default/classes/OneJNQuerySelector.cls:16]: Bulkify all the queries in this class

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/OneJNQuerySelector.cls [hunk 2]
```diff
@@ -13,31 +13,109 @@ public with sharing class OneJNQuerySelector {
...
+  public static List<Account> getAccounts(Set<Id> accountIdSet) {
+    List<Account> accountList = [
+      SELECT
+        RecordType.Name,
+        MonthsResidence_Number__c,
+        Political_Exposure__pc,
```
Comment (RemarioCRM) [inline force-app/main/default/classes/OneJNQuerySelector.cls:21]: i believe this field is wrong

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/OneJNQuerySelector.cls (hunk 2 above)
Comment (ishwari-gaikwad2) [reply force-app/main/default/classes/OneJNQuerySelector.cls:21]: which field?

Comment (RemarioCRM) [reply force-app/main/default/classes/OneJNQuerySelector.cls:21]: Political_Exposure__pc

File: force-app/main/default/classes/OneJNQuerySelector.cls [hunk 3]
```diff
@@ -13,31 +13,172 @@ public with sharing class OneJNQuerySelector {
...
+      SELECT
+        Professional_Category__c,
+        Position_List__c,
+        Industry__c,
+        Name,
+        FinServ__EmploymentStatus__c,
+        Time_Employed_in_Present_Employer_M__c,
+        Time_Employed_with_Previous_Employer_M__c,
+        Id
+      FROM FinServ__Employment__c
```
Comment (RemarioCRM) [inline force-app/main/default/classes/OneJNQuerySelector.cls:178]: add filter, current employment = TRUE

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

**Comment by RemarioCRM** [comment]
@ishwari-gaikwad2 and @truptizende-Thinqloud merge into 523

**Comment by RemarioCRM** [comment]
@ishwari-gaikwad2 and @truptizende-Thinqloud closing this pr. Changes are already in 529 branch.

**Pull request closed without merging**